src/ReadSpeedInput.cxx

```diff
@@ -0,0 +1,98 @@
+/* Copyright (C) 2020 Enrico Guiraud
+   See the LICENSE file in the top directory for more information. */
+
+#include "ReadSpeedInput.hxx"
+
+#include <iostream>
+#include <cstring>
+
+namespace ReadSpeed {
+
+void PrintThroughput(const Result &r)
```

Comment:
something called `PrintThroughput` does not seem to belong in a file called `ReadSpeedInput.cxx`. I would call this file `ReadSpeedArgParsing.cxx` or something like that and only put the argument parsing logic in here. what do you think?

Reply:
Could I maybe call it something like `ReadSpeedCLI`/`ReadSpeedIO`? That way I keep all the input parsing and output logging together which seems natural

Reply:
ReadSpeedCLI sounds good. Then let's be good people and also add a comment at the top of the header file that explains the purpose of the header contents, e.g. "This header contains helper functions for the root-readspeed CLI program, such as argument parsing and output formatting.".

IO is too overloaded in our context (and when you say IO my mind immediately goes to reading/writing data, which is in fact the most relevant "IO" also in the context of root-readspeed).

Reply:
Solid point on IO, I've gone with the CLI version and added a small comment for it